webui/react/src/components/ProjectCard.module.scss

```diff
@@ -1,32 +1,16 @@
 .base {
-  display: grid;
+  display: flex;
   gap: 16px;
-  grid-template:
-    'header  header' auto
-    'workspace workspace' 20px
-    'footer footer' 20px / auto auto;
-  height: 100%;
 
-  .headerContainer {
-    grid-area: header;
-  }
-  .name {
-    font-size: 13px;
-    margin-bottom: 0;
-    width: fit-content;
-  }
-  .workspaceContainer {
-    grid-area: workspace;
-  }
-  .workspaceIcon {
-    width: fit-content;
+  &.uncategorized .workspaceContainer {
+    visibility: hidden;
   }
   .footerContainer {
     align-items: end;
     display: flex;
+    flex-grow: 1;
     font-size: 12px;
     gap: 6px;
```

Comment:
nit: `var(spacing-sm)`

also nit: realizing we could just use `Row` here and eliminate more css, but also understand if we want to just get this PR done without adding another thing that's technically out of scope.

Reply:
✅